### PR TITLE
Feature/#10

### PR DESCRIPTION
**Pull request by Habiba67y**
Created Truck and TruckDto

### CODE REVIEW

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Training.TruckWorld.Backend.Domain.Common;
+
+namespace Training.TruckWorld.Backend.Domain.Entities
+{
+    public class ContactUser
+    {
+        public Guid UserId { get; set; }
```
Comment (Mirzohid-Mominov) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs:12]: ContactUser ni uzini  Id si yuq

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string EmailAddress { get; set; }
+        public string PhoneNumber { get; set; }
+
```
Comment (Mirzohid-Mominov) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs]: Constructor yozish kerak

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs (hunk 1 above)
Comment (Habiba67y) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs:12]: fixed

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs (hunk 2 above)
Comment (Habiba67y) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/ContactUser.cs]: fixed

**Review by Mirzohid-Mominov — APPROVED** [review]
not bad

**Review by Mirzohid-Mominov — APPROVED** [review]

**Review by Mirzohid-Mominov — APPROVED** [review]

**Review by Mirzohid-Mominov — APPROVED** [review]

**Review by Mirzohid-Mominov — APPROVED** [review]

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/Component.cs [hunk 3]
```diff
@@ -16,13 +16,13 @@ public class Component : SoftDeletedEntity
     public double Weight { get; set; }
     public ComponentCondition Condition { get; set; }
     public string Description { get; set; }
-    public Action Action { get; set; }
```
Comment (Mirzohid-Mominov) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Domain/Entities/Component.cs:20]: there is a conflict 
fix it

**Review by Mirzohid-Mominov — DISMISSED** [review]

**Review by Mirzohid-Mominov — APPROVED** [review]

**Review by Mirzohid-Mominov — APPROVED** [review]

**Review by azizbek-abdurahmonov — APPROVED** [review]

**Pull request merged**